xray/plot/plot.py

```diff
@@ -386,8 +390,12 @@ def _plot2d(plotfunc):
         The mapping from data values to color space. If not provided, this
         will be either be ``viridis`` (if the function infers a sequential
         dataset) or ``RdBu_r`` (if the function infers a diverging dataset).
-        When ``levels`` is provided and when `Seaborn` is installed, ``cmap``
-        may also be a `seaborn` color palette or a list of colors.
+        When when `Seaborn` is installed, ``cmap`` may also be a `seaborn` 
+        color palettes.
+    colors : a single color or a list of colors, optional 
+        When `Seaborn` is installed, ``colors`` may be a single color
```

Comment:
This should have some more general description, e.g., "Discrete colors to plot."